src/components/Main.jsx

```diff
@@ -0,0 +1,67 @@
+import style from './Main.module.css';
+import Person from '../../images/illustration-working.svg?react';
+import Brand from '../../images/icon-brand-recognition.svg?react';
+import Detailed from '../../images/icon-detailed-records.svg?react';
+import Fully from '../../images/icon-fully-customizable.svg?react';
```

Comment:
Looks like your images folder leaves in a wrong place. You shouldn't store any static assets in the root folder level. In some cases you might want to have a public folder and store your images there (like it was with a previous project), but most of the times it has to be an `assets` folder inside of `scr` since those images are static resources.